### ISSUE TITLE
How to add Decline button

### ISSUE THREAD

**Issue by duongthienlee** [Mastermindzh/react-cookie-consent#32]
It looks like there is only 1 button to accept the cookies.
How can I add a decline button?

**Comment by duongthienlee** [comment]
I got the idea, but when we click the decline button, the pop up should disappear as well.
I believe this can be done by  this.setState({ visible: false }) inside the cookies. But there is no props to change that

**Comment by duongthienlee** [comment]
Another question is that how can I access cookies after user accept it?
In my case I add  extraCookieOptions={{ "language": this.state.value }}

**Comment by Mastermindzh** [comment]
The cookiebar works like this:

1. check whether cookie is set, if not show the bar.
2. when the user clicks "accept" we create a cookie and hide the bar.

In order to hide the bar without a cookie you'd add a boolean into your class' state:
```jsx
this.state = {
    name: "React",
    showCookieBar: true
};
```

Then you render the cookie bar if showCookieBar is true like so:

```
{this.state.showCookieBar && (
    <CookieConsent></CookieConsent>
}
```

And finally you have to make the button toggle the boolean:

```jsx
onClick={() => {
    this.setState({ showCookieBar: false });
}}
```

This all results in the following code: [click here to open stackblitz](https://stackblitz.com/edit/react-cookie-consent-decline-button-demo)

## Accessing cookies

To access a cookie import the "Cookies" object from react-cookie-consent:

```jsx
import {Cookies} from "react-cookie-consent";
```

then use it like so:

```jsx
Cookies.get("cookieName");
```

In the [stackblitz](https://stackblitz.com/edit/react-cookie-consent-decline-button-demo) I have added an example for this as well.

**Comment by duongthienlee** [comment]
> The cookiebar works like this:
> 
> 1. check whether cookie is set, if not show the bar.
> 2. when the user clicks "accept" we create a cookie and hide the bar.
> 
> In order to hide the bar without a cookie you'd add a boolean into your class' state:
> 
> ```js-jsx
> this.state = {
>     name: "React",
>     showCookieBar: true
> };
> ```
> 
> Then you render the cookie bar if showCookieBar is true like so:
> 
> ```
> {this.state.showCookieBar && (
>     <CookieConsent></CookieConsent>
> }
> ```
> And finally you have to make the button toggle the boolean:
> 
> ```js-jsx
> onClick={() => {
>     this.setState({ showCookieBar: false });
> }}
> ```
> 
> This all results in the following code: [click here to open stackblitz](https://stackblitz.com/edit/react-cookie-consent-decline-button-demo)
> 
> ## Accessing cookies
> To access a cookie import the "Cookies" object from react-cookie-consent:
> 
> ```js-jsx
> import {Cookies} from "react-cookie-consent";
> ```
> 
> then use it like so:
> 
> ```js-jsx
> Cookies.get("cookieName");
> ```
> 
> In the [stackblitz](https://stackblitz.com/edit/react-cookie-consent-decline-button-demo) I have added an example for this as well.

Thank for your reply, but the Cookies.get("cookieName") function always return value true.  What I want is to store language preferences. So basically my cookies bar will contain a language switch, which will store language preferences. For example, the value should be a string "en" or "es" or "fi". I add extraCookieOptions object but don't know how to access that object

**Comment by Mastermindzh** [comment]
`Cookies.get("cookieName")` doesn't always return true, it'll return true if the cookie has the value of true.

In the [stackblitz example](https://stackblitz.com/edit/react-cookie-consent-decline-button-demo) you can clearly see it return "undefined", then we set it to 500 and finally we read it again and it returns 500.

Anyway, this no longer has anything to do with react-cookie-consent so I'll close the issue.

**Comment by AbdulAhad125** [comment]
@Mastermindzh  i had an question i am using this package but it is not storing data on accept button click 
Can you help me out

**Comment by AbdulAhad125** [comment]
I want to store cookie only after click

**Comment by Mastermindzh** [comment]
Check to see if you have "debug" listed as a prop.
What you're describing is the default behavior.

Please start a new issue instead of hijacking a totally unrelated issue :)
